### PR TITLE
[WIP] Remove 'build_dir_path' from 'tools/pip/setup.py'

### PR DESCRIPTION
**Pull request by klueska**
This variable is not referenced anywhere, and causes the setup.py script
to fail when it attempts to remove 'build_dir_path' since it was never
created in the first place.

### CODE REVIEW

File: tools/pip/setup.py [hunk 1]
```diff
@@ -92,8 +89,7 @@ def main():
     )
 
     # clean up build detritus:
-    shutil.rmtree(build_dir_path)
-    shutil.rmtree(os.path.join(script_dir, "{}.egg-info".format(output_name)))
+    shutil.rmtree(os.path.join(script_dir, "pip-egg-info", "{}.egg-info".format(output_name)))
```
Comment (elezar) [inline tools/pip/setup.py]: I don't see where the original path is set. Is this added when building the egg?

**Comment by klueska** [comment]
I'm still playing around with this and I'm not sure this is the full, right way to do things yet. I'll ping you once it's more stable. Is there a way to block CI from being triggered when I push an update to the branch?

**Comment by mpereira** [comment]
> Is there a way to block CI from being triggered when I push an update to the branch?

I think there's no way to do that right now, unfortunately.

**Comment by klueska** [comment]
I just decided to closse it for now to avoid triggering it. I'll create a new PR when I have something more credible.

**Pull request closed without merging**